data/lua/connector_ladx_bizhawk.lua

```diff
@@ -135,6 +136,10 @@ while true do
             udp:sendto(reply, msg_or_ip, port_or_nil)
         end
     end
+end
 
-    emu.frameadvance()
+event.onmemoryexecute(on_vblank, 0x40, "ap_connector_vblank")
+
+while true do
```

Comment:
The emu.yield() loop should not be needed here, and is probably just burning cycles since there is now an event callback registered. Is there a reason for this I'm missing?

Reply:
It's needed on my machine and should have minimal perf impact